scripts/noshake.json

```diff
@@ -348,6 +348,7 @@
   "Mathlib.Logic.Function.Defs": ["Mathlib.Tactic.AdaptationNote"],
   "Mathlib.Logic.Function.Basic": ["Batteries.Tactic.Init"],
   "Mathlib.Logic.Equiv.Defs": ["Mathlib.Data.Bool.Basic"],
+  "Mathlib.LinearAlgebra.VectorSpace": ["Mathlib.Algebra.Module.Pi"],
```

Comment:
(I'm aware this is needed because I used `Mathlib.Algebra.Module.Pi` -- my rationale was that someone looking for the alias was likely to be newer, and that this other import seems reasonably cheap anyways. Let me know if that doesn't seem like a good enough reason to ignore this.)

Reply:
Please add a test file with `variable? [VectorSpace K V]` and `guard_msgs` instead.

Reply:
(newbie:) Is there a standard format for adding a test file? What to name it, where to place it? Just asking so I can mimick the moves in my own PR.